SPECS/iptables/iptables.spec

```diff
@@ -73,13 +76,14 @@ find %{buildroot} -name '*.la' -delete
 
 %clean
 rm -rf %{buildroot}/*
+
```

Comment:
@thcrain-msft linter seems angry about this line. Is this expected?

Reply:
The linter's anger is not justified- I'm not sure why it's asking for the extra line there. I'll update the tooling, feel free to merge without spec-lint passing.